server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ContestDownloadByID.java

```diff
@@ -25,7 +28,7 @@
  * @version 0.0.1
  */
 @SuppressWarnings("PMD.AtLeastOneConstructor")
-public class ContestDownloadByID extends AbstractEndpoint {
+public class ContestDownloadByID extends AbstractCountyDashboardEndpoint {
```

Comment:
This endpoint might also need to be accessed by the state dashboard to get contest information for display.